packages/css/src/components/form/form-group.scss

```diff
@@ -54,11 +55,15 @@
   legend.dso-label-container {
     border-bottom: 0;
     font-size: $font-size-base;
-    margin-bottom: $u1;
+    margin-bottom: 0;
   }
 
   legend + .dso-label-container {
-    margin-bottom: $u1;
+    margin-bottom: 0;
+  }
+
+  .dso-label-container + .dso-field-container {
+    margin-top: $u1;
```

Comment:
Waar is deze margin-top voor nodig?

Reply:
Die is nodig om ruimte te creëren tussen het label en het veld in verticale formulieren (of smalle horizontale formulieren)

Reply:
OK!